bayesian/api_v1.py

```diff
@@ -1205,6 +1205,21 @@ def get(ecosystem):
         return jsonify(result)
 
 
+class CvesByDateEcosystem(ResourceWithSchema):
```

Comment:
You should not create this class in api_v1.py file. Please create a new file under `bayesian` directory